dhis-2/dhis-test-integration/src/test/resources/tracker/programrule/event_with_last_date.json

```diff
@@ -0,0 +1,79 @@
+{
+  "importMode": "COMMIT",
```

Comment:
This file is parsed into `TrackerObjects` so should it not only contain

```java
public class TrackerObjects implements Serializable {
  @JsonProperty @Builder.Default
  private final List<TrackedEntity> trackedEntities = new ArrayList<>();
  @JsonProperty @Builder.Default private final List<Enrollment> enrollments = new ArrayList<>();
  @JsonProperty @Builder.Default private final List<Event> events = new ArrayList<>();
  @JsonProperty @Builder.Default private final List<Relationship> relationships = new ArrayList<>();
}
```

?

This would make our files easier to read as we mostly put in fields that are ignored anyway like `createdAt`, ... 😅

Reply:
Done